src/runner/__tests__/runESLint.test.js

```diff
@@ -15,7 +15,20 @@ const runESLintRunnerWithMockedEngine = ({
 
       lintFiles() {
         return mockOptions.errorCount > 0
-          ? [{ errorCount: mockOptions.errorCount, warningCount: 0 }]
+          ? [
+              {
+                errorCount: mockOptions.errorCount,
+                warningCount: 0,
+                messages: [
```

Comment:
The number of entries here should probably match `mockOptions.errorCount` but for the purposes of testing, this is probably fine? The original test didn't even have `messages` to begin with.